content/en/contribute/medic/product-development-process/focused-groups.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Focused Groups"
+linkTitle: "Focused Groups"
+weight: 1
+description: >
+  How Product Team is structured to better serve the community
```

Comment:
These groups were originally called Focused Working Groups internally. They are sometimes called Focused Groups as shorthand in some contexts, but I think we should maintain the full term in these pages because they are more like _"working groups"_ (a generally known thing that is similar to what we have), than _"focus groups"_ (also a known thing, but completely different).

```suggestion
title: "Focused Working Groups"
linkTitle: "Focused Working Groups"
weight: 1
description: >
  Cross-functional teams designed to better serve the community
```